### ISSUE TITLE
sf::Texture::loadfromfile() Not working

### ISSUE THREAD

**Issue by KierenP** [KierenP/Heist#6]
in commit fc9d7f701100fa2b5aadc0a1198e49a6851ff08b:

```
if (!MyTexture.loadFromFile("C:\\Users\\Kieren\\Documents\\GitHub\\Heist\\PlaceHolderPlayer.png"));
    return -1;
```

`MyTexture.loadFromFile` always fails to load the texture and `return -1;` always executes. No leads as to why.

**Comment by KierenP** [comment]
Its because there's a semi colon on the if statement retard

**Comment by KierenP** [comment]
Issue to be closed when I commit code from Week 2 Mondays lesson this afternoon.

**Comment by KierenP** [comment]
Fixed in commit 3f7c3df94074aa87e6c96f004f9bb95c739c0a10.
